changelog/2516.feature

```diff
@@ -0,0 +1 @@
+store unmeshed marks on functions pytestmark attribute
```

Comment:
This could be more descriptive to end users:

```
Now test function objects have a ``pytestmark`` attribute containing a list of marks applied directly to the test function, as opposed to marks inherited from parent classes or modules.
```

Also we should definitely document this properly somewhere on the docs and make it official part of the API.